templates/publisher/account-snaps.html

```diff
@@ -0,0 +1,166 @@
+        <p>Electron, Python, Go and many more examples of how to 'snap' an application using familiar languages.</p>
+        <p><a href="https://docs.snapcraft.io/build-snaps/languages" class="p-button--neutral">Choose a language guide</a></p>
+      </div>
+    </div>
+  </div>
+  <div class="p-strip is-shallow">
+    <div class="row p-divide">
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Snapcraft documentation</h3>
+        <p>All you need to know about snaps, snapcraft and everything in between.</p>
+        <p><a href="https://docs.snapcraft.io">Read the docs &rsaquo;</a></p>
+      </div>
+      <div class="col-6 p-divider__block">
+        <h3 class="p-heading--five">Join the community</h3>
+        <p>Request features, troubleshoot and generally find all the latest talk about snaps, snapd and snapcraft.</p>
+        <p><a href="https://forum.snapcraft.io">Go to the focum &rsaquo;</a></p>
+      </div>
+    </div>
+  </div>
+{% else %}
+  <div class="p-strip is-shallow u-no-padding--top">
+    <div class="row u-no-margin--top">
+      <table>
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Privacy</th>
+            <th>Owner</th>
+            <th>Latest release</th>
+            <th>Version</th>
+          </tr>
+        </thead>
+        <tbody>
+          {% for snap in snaps|sort %}
+            <tr>
+              <td>
+                <a href="/account/snaps/{{snap}}/market" class="u-no-margin--top u-vertically-center">
+                  {% if snaps[snap].icon_url %}
+                    <img src="{{ snaps[snap].icon_url }}" class="p-snap-list__image" />
+                  {% else %}
+                    <img src="https://assets.ubuntu.com/v1/6fbb3483-snapcraft-default-snap-icon.svg" class="p-snap-list__image" />
+                  {% endif %}
+                  <span class="p-snap-list__name u-no-margin--top">
+                    {{snap}}
+                  </span>
+                </a>
+              </td>
+              <td>
+                {% if snaps[snap].private %}
+                  Private
+                {% else %}
+                  Public
+                {% endif %}
+              </td>
+              <td>
+                {% if snaps[snap].publisher.username == current_user %}
+                  You
+                {% else %}
+                  {{ snaps[snap].publisher['display-name'] }}
+                {% endif %}
+              </td>
+              {% if snaps[snap].latest_revisions[0].status == 'Published' %}
+                <td>{{ snaps[snap].latest_revisions[0].channels[0] }}</td>
+              {% else %}
+                <td>Not released</td>
+              {% endif %}
+              <td>{{ snaps[snap].latest_revisions[0].version }}</td>
+            </tr>
+          {% endfor %}
+        </tbody>
+      </table>
+    </div>
+    {% if snaps|length == 1 %}
+      {% for snap in snaps %}
+        <div class="p-strip--light is-shallow">
```

Comment:
This is a p-strip inside another p-strip. It's a bit of a weird nesting and it causes that there is no spacing between the table and this strip.
<img width="1702" alt="screen shot 2018-04-13 at 11 31 55" src="https://user-images.githubusercontent.com/83575/38727795-4f152e4a-3f0e-11e8-9ae0-b574fb40c6ef.png">

Moving this strip one step higher in DOM tree fixes this issue:

<img width="1720" alt="screen shot 2018-04-13 at 11 33 02" src="https://user-images.githubusercontent.com/83575/38727857-76d5d0d8-3f0e-11e8-83a1-6f8c02ad0358.png">